docs/worker-node/install-singularity.md

```diff
@@ -38,7 +39,7 @@ they enable it to run unprivileged.
     lower security risk.
 
 The document is intended for system administrators who wish to either
-enable singularity to be run as an unprivileged user, install privileged
+enable singularity to be run as an unprivileged user, install
```

Comment:
Maybe we should be more general and say this doc is intended for sysadmins who wish to install or configure singularity.

Reply:
ok